### DISCUSSION TITLE
Hydrogen gets out of sync when Ardour is jack master.

### DISCUSSION THREAD

**Discussion by dchurch24** [hydrogen-music/hydrogen#1729] [Hydrogen on Linux]
Hi. As per the title, when using Jack with Hydrogen, the interface appears to get out of sync with the actual audio. This is version 1.1.1. On 22.04.2.

I first noticed it when I created a pattern with one kick drum for every beat, pressed the play button and it misses the first beat. Once the pattern repeats that first beat sounds.

It also misses it when exporting to a wav file, it's like it's just not there the first time around the pattern.

If I disconnect from Jack, then all the beats sound.

I've uninstall and reinstalled, but the problem persists.

Anyone got any clues?

**Comment by theGreatWhiteShark** [comment]
Hey @dchurch24 ,

This issue should be already fixed in the [latest release](https://github.com/hydrogen-music/hydrogen/releases/tag/1.2.0-beta1).

**Reply by dchurch24** [reply]
Hi. Thank you :)

After some initial searching and trying a few things out (none of which worked - matching time sigs, covering blanks in the song etc...), I simply made Hydrogen the Jack Master (about 10 minutes ago!) - and that seems to have solved it. 

I'll install the latest release, thank you.

Keep up the excellent work, it's appreciated :)

**Reply by theGreatWhiteShark** [reply]
> I simply made Hydrogen the Jack Master (about 10 minutes ago!) - and that seems to have solved it.

It may still be a pain since some other applications are not happy about not being registered as master themselves. They either routinely try to register again or still think internally they are and do not listen to the tempo information send by Hydrogen.

Personally, I tend to deactivate the JACK TBM support in the Hydrogen's Preferences.

**Comment by dchurch24** [comment]
Hi again...is there an update to that tar file? I'm trying to install it but the README.txt says "read the install.txt file" - which is nowhere to be found.

**Reply by trebmuh** [reply]
https://github.com/hydrogen-music/hydrogen/blob/master/INSTALL.md

**Reply by dchurch24** [reply]
Thank you. You're a gentleman and scholar!
